hack/e2e.sh

```diff
@@ -54,29 +54,40 @@ oc secrets link default ci-pull-secret --for=pull
 # `oc new-app` would stumble on unknown monitoring.coreos.com/v1 objects, so process and create instead
 oc process -f dist/openshift/observability.yaml -p NAMESPACE="cincinnati-e2e" | oc apply -f -
 
+# Export the e2e test environment variables
+E2E_TESTDATA_DIR="${E2E_TESTDATA_DIR:-graph-builder/tests/e2e/testdata}"
+export E2E_TESTDATA_DIR
+read -r E2E_METADATA_REVISION <"${E2E_TESTDATA_DIR}"/metadata_revision
+export E2E_METADATA_REVISION
+
 # Apply oc template
 oc new-app -f dist/openshift/cincinnati.yaml \
   -p IMAGE="${IMAGE}" \
   -p IMAGE_TAG="${IMAGE_TAG}" \
   -p GB_CPU_REQUEST=50m \
   -p PE_CPU_REQUEST=50m \
   -p RUST_BACKTRACE="1" \
-  -p GB_PLUGIN_SETTINGS='
+  -p GB_PLUGIN_SETTINGS="$(cat <<-EOF
```

Comment:
bash :/

Lets use the following:

* create a plain param file:
```
IMAGE="${IMAGE}"
IMAGE_TAG="${IMAGE_TAG}"
...
```

* use `sed`/`envsubst`/moar bash magic to template it
* run `oc new-app -f dist/openshift/cincinnati.yaml -p param-file`

Reply:
I'd rather not rework this now as part of this PR because it works as is. I do agree that we want to change this in the future, but refactoring it is out of this PRs scope.

Reply:
okay, lets rework it later once we introduce different e2e tests